src/peergos/shared/io/ipfs/multihash/Multihash.java

```diff
@@ -35,8 +39,9 @@ public static Type lookup(int t) {
     }
 
     public final Type type;
-    private final byte[] hash;
+    public final byte[] hash;
```

Comment:
I don't want this public because it is mutable

Reply:
fixed